actors/evm/src/interpreter/instructions/stack.rs

```diff
@@ -66,3 +67,11 @@ fn test_push_pad_right() {
     assert_eq!(stack.len(), 1);
     assert_eq!(stack.pop().unwrap(), U256::from(0xdead0000u64));
 }
+
+#[test]
+fn test_push0() {
+    let mut stack = Stack::new();
+    assert_eq!(push::<0>(&mut stack, &[0x1]).unwrap(), 0);
```

Comment:
Would probably saturate the full word (0xff..ff) for a better assert, since this value anyway contains zeros once padded to a word.